lib/network.sh

```diff
@@ -76,7 +77,11 @@ fi
 # Calculate DHCP range
 network_address dhcp_range_start "$PROVISIONING_NETWORK" 10
 network_address dhcp_range_end "$PROVISIONING_NETWORK" 100
+network_address PROVISIONING_POOL_RANGE_START "$PROVISIONING_NETWORK" 100
+network_address PROVISIONING_POOL_RANGE_END "$PROVISIONING_NETWORK" 200
```

Comment:
Are these ranges only for the cases where user does not define them?

Reply:
no, they are always defined